ui/app/pages/unlock-page/unlock-page.component.js

```diff
@@ -136,7 +136,7 @@ export default class UnlockPage extends Component {
         style={style}
         disabled={!this.state.password}
         fullWidth
-        variant="raised"
+        variant="contained"
```

Comment:
`raised` was an old, now non supported value for the `variant` type. `contained` is appropriate here, no visible regression: https://material-ui.com/api/button/